Core/Logging/AnalyticEvent.swift

```diff
@@ -32,23 +30,17 @@ public struct AnalyticEvent: Codable {
     ) {
         self.type = type
         self.action = action
-        self.new = new
-        self.old = old
+        self.new = Self.escape(new)
+        self.old = Self.escape(old)
         self.version = version
         self.timestamp = timestamp
     }
 
-    private func escape(_ dictionary: [String: String]?) -> String? {
-        let encoder = JSONEncoder()
-        guard
-            dictionary != nil,
-            let dictionaryString = try? encoder.encode(dictionary),
-            let jsonString = String(data: dictionaryString, encoding: .utf8)
-        else {
-            return nil
-        }
-        return NSRegularExpression.escapedPattern(for: jsonString).replacingOccurrences(of: "\\", with: "")
+    private static func escape(_ dictionary: [String: Any]?) -> String? {
+        guard let dictionary = dictionary,
+              let data = try? JSONSerialization.data(withJSONObject: dictionary, options: []) else { return nil }
+        return String(data: data, encoding: String.Encoding.utf8)
```

Comment:
Could we omit `String.Encode` and return `String(data: data, encoding: .utf8)` ?

Reply:
✅